### PR TITLE
DOC-1045, DOC-1046: Updated ALTER COLUMN SQL diagram

### PR DESCRIPTION
**Pull request by nickvigilante**
Release note: None

Fixes DOC-1045, DOC-1046

### CODE REVIEW

File: pkg/storage/sst_iterator.go [hunk 1]
```diff
@@ -33,6 +33,10 @@ type sstIterator struct {
 
 	// roachpb.Verify k/v pairs on each call to Next.
 	verify bool
+
+	// For determining whether to trySeekUsingNext=true in SeekGE.
+	prevSeekKey  MVCCKey
```
[the remark this answers is not in the document]
Comment (nickvigilante) [reply pkg/storage/sst_iterator.go:38]: You're absolutely right. Lemme remove those.

**Comment by RichardJCai** [comment]
I think there are a bunch of extraneous files / changes checked in here, only `diagrams.go` and the corresponding generated files should be changed right?

**Comment by nickvigilante** [comment]
Correct. I'm fixing it now. I accidentally used wildcards despite there only being a few changes in git add.

**Review by RichardJCai — APPROVED** [review]

**Comment by RichardJCai** [comment]
Oh actually can you run `make generate`, seems like some generated files are out of date according to CI

**Comment by nickvigilante** [comment]
I ran `sudo make generate` locally and then force pushed the branch again. Are there any files from here I'm missing? This is what I see when I run `git status`:

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	.idea/
	c-deps/protobuf/
	pkg/sql/lex/keywords.go
	pkg/sql/lex/tokens.go
	pkg/ui/ccl/
	pkg/ui/src/

**Comment by nickvigilante** [comment]
@RichardJCai Mind taking another look at this?

**Review by ajwerner — COMMENTED** [review]
> Reviewed 1 of 11 files at r3.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/74767)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @nickvigilante, @postamar, and @RichardJCai)

___
*[pkg/cmd/docgen/diagrams.go, line 372 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/74767#-Mzor76x6-ZTHT6SRFEU:-Mzor76y8j4z_JMlt9Up:bm28xw6) ([raw file](https://github.com/cockroachdb/cockroach/blob/7e2e87051c3af185557a76c96b6946b69817b6ab/pkg/cmd/docgen/diagrams.go#L372)):*
> ```Go
> 		replace: map[string]string{
> 			"relation_expr": "table_name",
> 			"alter_column_visible": "'SET' ( 'NOT' | ) 'VISIBLE'",
> ```

does the `|` need to be here? What does it mean?

**Review by nickvigilante — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/74767)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ajwerner, @postamar, and @RichardJCai)

___
*[pkg/cmd/docgen/diagrams.go, line 372 at r3](https://reviewable.io/reviews/cockroachdb/cockroach/74767#-Mzor76x6-ZTHT6SRFEU:-Mzp08Q_2C-TfubQWlN9:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/7e2e87051c3af185557a76c96b6946b69817b6ab/pkg/cmd/docgen/diagrams.go#L372)):*
<details><summary><i>Previously, ajwerner wrote…</i></summary><blockquote>

does the `|` need to be here? What does it mean?
</blockquote></details>

Done.

**Review by ajwerner — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/74767#-:-Mzp0k4z0zI_NToYGyon:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @ajwerner, @postamar, and @RichardJCai)

**Comment by nickvigilante** [comment]
@ajwerner Can you BORS for me? I don't have access since I'm not in Eng.

**Comment by ajwerner** [comment]
@nickvigilante what's the status of this?

**Comment by nickvigilante** [comment]
@ajwerner I'm closing this. This should be restarted from scratch. I'm gonna work with Stephanie on Thursday for stuff with diagrams.

**Pull request closed without merging**